modules/mobfoxBidAdapter.js

```diff
@@ -80,103 +71,63 @@ function MobfoxAdapter() {
       n_rating_req: bidParams.n_rating_req || undefined
     };
 
-    return requestParams;
-  }
+    let payloadString = buildPayloadString(params);
 
-  function sendBidRequest(bid) {
-    let requestParams = buildBidRequest(bid);
-    let queryString = buildQueryStringFromParams(requestParams);
-
-    ajax.ajax(`${BID_REQUEST_BASE_URL}?${queryString}`, {
-      success(resp, xhr) {
-        if (xhr.getResponseHeader('Content-Type') == 'application/json') {
-          try {
-            resp = JSON.parse(resp)
-          } catch (e) {
-            resp = {error: resp}
-          }
-        }
-        onBidResponse({
-          data: resp,
-          xhr: xhr
-        }, bid);
-      },
-      error(err) {
-        if (xhr.getResponseHeader('Content-Type') == 'application/json') {
-          try {
-            err = JSON.parse(err);
-          } catch (e) {
-          }
-          ;
-        }
-        onBidResponseError(bid, [err]);
+    return {
+      method: 'GET',
+      url: BID_REQUEST_BASE_URL,
+      data: payloadString,
+      requestId: bid.bidId
+    };
+  },
+  interpretResponse: function (serverResponse, bidRequest) {
+    const bidResponses = [];
+
+    if (!serverResponse || serverResponse.error) {
```

Comment:
Its LGTM except this one change.
I think you missed to update this condition. Your can access error response by `serverResponse.body.error`